Sources/PointFree/Episode/EpisodeModelExtensions.swift

```diff
@@ -0,0 +1,23 @@
+import EpisodeModels
+
+extension Episode {
+  var slug: String {
+    return "ep\(self.sequence)-\(PointFree.slug(for: self.title))"
```

Comment:
keeping the slug computation in this repo since it's domain specific and determines how our permalinks look. seems like it should stay local to this code.